pipeline_dp/budget_accounting.py

```diff
@@ -172,19 +174,21 @@ def compute_budgets(self):
 
         total_weight_eps = total_weight_delta = 0
         for mechanism in self._mechanisms:
-            total_weight_eps += mechanism.weight
-            if mechanism.mechanism_spec.use_delta():
-                total_weight_delta += mechanism.weight
+            for i in range(mechanism.mechanism_spec.count):
+                total_weight_eps += mechanism.weight
```

Comment:
Nit:
 total_weight_eps += mechanism.weight*count

and the same with delta

and no need to have for